src/views/Public/Pages/Records/View.vue

```diff
@@ -8,10 +11,6 @@
       {{ $t('record.recordDeleted') }}
     </b-alert>
 
-    <portal to="topbar-title">
-      {{ title }}
```

Comment:
You removed this, so you need to fix it. Plus the titles that Katrin implemented need to be used in the modal too!  So probably make a similar computed as the record toolbar portal one.

Reply:
I think I do not need portal here because I added the title in the modal directly. In the modal, component I also added the computed property.

Reply:
It does not work with Katrins changes, it should say Creating record of, instead of Record page for